src/components/product/OutOfStock/out-of-stock.scss

```diff
@@ -5,19 +5,19 @@
   flex-direction: column;
   margin-top: var(--fs-spacing-4);
 
-  .text__title-subsection {
+  [data-out-of-stock-title] {
```

Comment:
Since these comes from the ui package, shouldn't they have `data-store` on them?

Reply:
How this is an "inside component" it uses just [data-name-of-component], like [this](https://github.com/vtex/faststore/pull/1093)

Reply:
Thanks!